src/automations/components/AutomationActionSendNotificationInput.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <p-content class="automation-action-send-notification">
+    <p-label label="Block" :state="blockDocumentState" :message="blockDocumentError">
+      <BlockCapabilityBlockDocumentInput v-model="blockDocumentId" :state="blockDocumentState" capability="notify" required use-modal />
+    </p-label>
+
+    <template v-if="isAnonymousBlock">
+      <p-message info>
+        This notification action is using an anonymous block. This is most likely because it was migrated from an existing notification.
```

Comment:
```suggestion
        This notification action is using an anonymous block, most likely because it was migrated from an existing notification.
```
Is there call to action here? I'm not sure what purpose this is serving otherwise

Reply:
No not really. I'm trying to see if I can remove this. But I believe this is here because it'll show the block but its not one of the values in the list of regular blocks.